### DISCUSSION TITLE
i cant find any taps on brew i dont know why

### DISCUSSION THREAD

**Discussion by neouks** [Homebrew/discussions#1229] [Casks]
![image](https://user-images.githubusercontent.com/26845888/114117579-92bca200-9919-11eb-9cfa-0a040fc53761.png)
![image](https://user-images.githubusercontent.com/26845888/114117593-99e3b000-9919-11eb-8bcf-60d1a158dda8.png)
![image](https://user-images.githubusercontent.com/26845888/114117607-a10abe00-9919-11eb-840e-a3997534e0fb.png)
![image](https://user-images.githubusercontent.com/26845888/114117642-b5e75180-9919-11eb-81d4-978dbc028c31.png)

**Comment by gromgit** [comment]
In future, please post text output _as text_ (i.e. copy-n-paste), not screenshots. Images of text are really inconvenient to deal with.

It looks like your core tap is empty, so I'm not sure how you managed to get an "Installation successful" message. Do you still have the output of the installation process itself? If so, please post it here.

Also, what's the output of `brew update`?

**Reply by neouks** [reply]
sorry for that screenshot 
```
➜  ~ brew update
fatal: Could not resolve HEAD to a revision
Already up-to-date.
➜  ~
```

**Reply by gromgit** [reply]
Hmmm. Easiest thing to do at this point is to wipe and re-tap:
```
rm -fr $(brew --repo homebrew/core)
brew tap homebrew/core
```
then try your `brew search` again.

**Reply by neouks** [reply]
ok  solved!👍  thx bro love u 😘
